docs/model_server_rest_api_chat.md

```diff
@@ -0,0 +1,58 @@
+# OpenAI API {#ovms_docs_rest_api_chat}
+
+
+
+
+## API Reference
+Please see the [OpenAI API Reference](https://platform.openai.com/docs/api-reference/chat) for more information on the API. 
+
+### Request parameters
+We support now the following parameters:
+- model - this name is assigning the request to a graph of the same name
+- messages - user content type is supported as text. Currently it is not allowed to send the image_url in the user message content
+- frequency_penalty
+- max_tokens
+- presence_penalty
+- response_format - only json is implementation
+- stream
+- temperature
+- top_p
```

Comment:
Can we comment out the ones that are not supported currently on main and uncomment as changes that enable them are merged?